includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1503,16 +1503,29 @@ public function get_wizard_in_cart_payment_gateways() {
 		}
 
 		if ( $spotlight ) {
```

Comment:
I actually thought about redoing the whole logic there, but since this is a fix release I wanted to keep the changes minimal as to avoid a possible logic error. We can revisit this in 3.7 also with a refactor. IMO we should avoid doing refactors in fix releases.

Reply:
Fair enough. I will create a PR with this change and the removal of the images and tag it for 3.7.